client/my-sites/plans/legacy-plan-notice.jsx

```diff
@@ -25,6 +25,7 @@ const maybeRenderLegacyPlanNotice = ( eligibleForProPlan, { plan } ) => {
 			></Notice>
 		);
 	}
+	return null;
```

Comment:
Change looks good, but it seems like this function's JSDoc is missing a return type?

Reply:
True. Interestingly, this is still valid, because the rule we're fixing `require-returns-check` just means that "if `@returns` is in jsdoc, there must be a return statement in the code," not necessarily anything about the types.

I did change this to `import('react').Element`, which we seem to use elsewhere